.github/workflows/publish-documentation.yml

```diff
@@ -18,12 +18,10 @@ jobs:
       - name: Execution permission for Gradle wrapper
         run: chmod +x gradlew
       - name: Dokka
-        run: ./gradlew dokka
-      - name: Gather modules documentation
-        run: ./gradlew gatherDokkaDocumentation
+        run: ./gradlew dokkaHtmlMultiModule
       - name: Publish GitHub pages
         uses: JamesIves/github-pages-deploy-action@releases/v3
         with:
           ACCESS_TOKEN: ${{ secrets.accessToken }}
           BRANCH: gh-pages
-          FOLDER: build/dokka
+          FOLDER: build/dokka/html
```

Comment:
The built documentation is actually at `build/dokka/htmlMultiModule` and lacks an `index.html` page.

Reply:
Since we'll be moving to the `dokkaHtmlCollector` task, the folder will be `build/dokka/htmlCollector`.